develop 分支编译报错

编译分支: ```develop```

编译docker环境: 
```
paddlepaddle/paddle_manylinux_devel:cuda8.0_cudnn5
```

编译命令:
```
bash paddle/scripts/paddle_build.sh build
```

编译错误:
```
Scanning dependencies of target device_context
[  7%] Building CXX object paddle/fluid/platform/CMakeFiles/device_context.dir/device_context.cc.o
[  7%] Building CXX object paddle/fluid/platform/CMakeFiles/device_context.dir/init.cc.o
In file included from /paddle/paddle/fluid/framework/operator.h:25:0,
                 from /paddle/paddle/fluid/platform/init.cc:19:
/paddle/paddle/fluid/framework/attribute.h:23:49: fatal error: paddle/fluid/framework/framework.pb.h: No such file or directory
 #include "paddle/fluid/framework/framework.pb.h"
                                                 ^
compilation terminated.
```

--- a/paddle/fluid/platform/CMakeLists.txt
+++ b/paddle/fluid/platform/CMakeLists.txt
@@ -46,7 +46,7 @@ ENDIF()
 # memcpy depends on device_context, here add deps individually for
 # avoiding cycle dependencies
 cc_library(device_context SRCS device_context.cc init.cc DEPS malloc
-    place eigen3 stringpiece cpu_helper ${GPU_CTX_DEPS} ${MKLDNN_CTX_DEPS})
+    place eigen3 stringpiece cpu_helper framework_proto ${GPU_CTX_DEPS} ${MKLDNN_CTX_DEPS})
 nv_test(device_context_test SRCS device_context_test.cu DEPS device_context gpu_info)
 
 cc_test(init_test SRCS init_test.cc DEPS device_context)

